README.md

```diff
@@ -8,8 +8,8 @@ $ source /opt/ros/indigo/setup.bash
 $ mkdir -p ~/catkin_ws/src
 $ cd ~/catkin_ws   
 $ wstool init src
-$ wstool set robot-programming https://github.com/jsk-enshu/robot-programming --git -t src
-$ wstool set dynamixel_urdf    https://github.com/jsk-enshu/dynamixel_urdf    --git -t src
+$ wstool set robot-programming https://github.com/jsk-enshu/robot-programming --git -u -t src
+$ wstool merge -t src src/robot-programming/enshu.rosinstall
```

Comment:
if you use .rosinstall file, you will not need  'set robot-programming'

Reply:
Thank you for review.  Modification: https://github.com/jsk-enshu/robot-programming/pull/236/commits/ee7f13e5b9f82c8b226fac84e3d8cb4635a21dcc